### DISCUSSION TITLE
Can I have a comment after header_attributes ?

### DISCUSSION THREAD

**Discussion by reagle** [jgm/pandoc#10354] [Q&A] [answered]
I sometimes use an emdash for small section breaks; I also like to keep a word count of the size of the section in a html/markdown comment. But the latter interferes with the parsing of the former.

```markdown

## --- {style="text-align: center"} <!-- 899 words -->

```

yields

```html
<h2 id="styletext-align-center">— {style=“text-align: center”} </h2>

```

instead of:

```html
<h2 style="text-align: center" id="section-1">—</h2>

```

**Comment by jgm** [comment, marked as answer]
I think you got your answer: you can't.  The docs are clear that the header attribute has to come at the end of the line.

**Reply by reagle** [reply]
I thought there might be an exception when we use `--strip-comments` but that is not the case.
